scripts/bert/finetune_classifier.py

```diff
@@ -325,8 +330,37 @@ def evaluate(dataloader_eval, metric):
     logging.info(metric_str, *metric_val)
 
 
+def log_train(batch_id, batch_num, metric, step_loss, log_interval, epoch_id, learning_rate):
+    metric_nm, metric_val = metric.get()
```

Comment:
Although this is just a small auxiliary function, it is more standard if you can add a function comment.

In addition, `log_train` and `log_inference` currently appear to be slightly different only when constructing a log string. Is it better to integrate as a function or to keep it so separated now?

Reply:
Thank you @kenjewu . Doc string is added for each function. I would like to keep the functions separated as is.